### DISCUSSION TITLE
Spectrum OLED stopped working after latest update (20220116)

### DISCUSSION THREAD

**Discussion by pepper7777** [rern/rAudio#763] [Fixed Issues]
I have RPi3 with SH1106 SPI OLED display.
After applying latest update 20220116 the display stays on +R logo and never changes to audio level spectrum.
Any suggestions please?

Thanks
Pepper

**Comment by apastuszak** [comment]
What does the display look like on these while music is playing?

**Reply by pepper7777** [reply]
![spectrum](https://user-images.githubusercontent.com/53092601/149690657-de08c0fe-b4eb-402d-80c1-6776cbd2cf81.jpg)

It supposed to look like that:
https://raw.githubusercontent.com/rern/rAudio-1/main/srv/http/assets/img/mpdoled.jpg

**Comment by apastuszak** [comment]
Can you hook up 2 of those, one for each channel?

**Comment by Dave0616** [comment]
Have you been able to find a fix for this im having the same issue

**Comment by Dave0616** [comment]
> I have RPi3 with SH1106 SPI OLED display. After applying latest update 20220116 the display stays on +R logo and never changes to audio level spectrum. Any suggestions please?
> 
> Thanks Pepper

https://github.com/rern/rAudio/discussions/1797
